src/framework/global/internal/interactive.h

```diff
@@ -29,12 +29,14 @@
 #include "ui/imainwindow.h"
 
 #include "../iinteractive.h"
+#include "shortcuts/ishortcutsregister.h"
 
 namespace muse {
 class Interactive : public IInteractive, public Injectable, public async::Asyncable
 {
     Inject<muse::ui::IInteractiveProvider> provider{ this };
     Inject<muse::ui::IMainWindow> mainWindow{ this };
+    Inject<shortcuts::IShortcutsRegister> shortcutsRegister = { this };
```

Comment:
copied & pasted from elsewhere, so it seems we've used two different conventions for initialising - should I change them all? Or just this one to match this file?

Reply:
> should I change them all
yes, please

Reply:
done